randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -963,19 +963,26 @@ Extra - minimap_data: [{'x': 70, 'y': 44}, {'x': 70, 'y': 45}, {'x': 70, 'y': 46
   * Extra - instance_id: 134710
   > Door to Bullet Hell Room
       Trivial
+  > EMP Ball
+      Any of the following:
+          Morph Ball and Morph Glide (Advanced)
+          Any of the following:
+              # Shoot the EMP from the door
+              Super Missiles
+              # RNG dependant
+              Missiles and Knowledge (Expert)
 
 > Door to Bullet Hell Room; Heals? False
   * Layers: default
   * Distribution Center Bullet Hell Room Access EMP Door to Bullet Hell Room/Door to Bullet Hell Room Access
   * Extra - instance_id: 134711
-  > Door to Distribution Facility Intersection
+  > Center
       Any of the following:
           Space Jump or Enabled Septogg Helpers or Can Use Spider Ball
-          Speed Booster and Shinesparking Tricks (Beginner)
           # IBJ
           Infinite Bomb Jumping (Expert) and Can Use Bombs
-  > Event - EMP
-      Trivial
+          # Shinespark before door to Bullet Hell
+          Speed Booster and Shinesparking Tricks (Beginner) and Short Charge (Intermediate)
```

Comment:
AFAIK Short charging is not needed?

Reply:
Short Charge (crouching) was needed. Especially if the door to Bullet Hell Room is one the player can't open.

Reply:
Can you add an OR with "no entrance rando + no dock rando + emp event done" as an alternative then please?